src/framing/FrameTransport.cpp

```diff
@@ -149,9 +149,15 @@ void FrameTransport::onComplete() {
   terminateProcessor(folly::exception_wrapper());
 }
 
-void FrameTransport::onError(std::exception_ptr ex) {
-  VLOG(6) << "onError" << folly::exceptionStr(ex);
-  terminateProcessor(folly::exception_wrapper(std::move(ex)));
+void FrameTransport::onError(std::exception_ptr eptr) {
+  VLOG(6) << "onError" << folly::exceptionStr(eptr);
+
+  try {
```

Comment:
I believe the input "eptr" should never be nullptr, but in case, should we check if eptr is null?

http://en.cppreference.com/w/cpp/error/rethrow_exception